docs/about/features_index/fed_eval.rst

```diff
@@ -42,468 +42,148 @@ With OpenFL version==1.7 aggregator start command is enhanced to have an optiona
-
-                            collaborator_count: 2                                                                      
-
-                        template: src.dataloader.PyTorchMNISTInMemory                                                
-
-                        network:                                                                                       
-
-                        settings:                                                                                    
-
-                            agg_addr: devvm###.com                                                
-
-                            agg_port: 55529                                                                            
-
-                            cert_folder: cert                                                                          
-
-                            client_reconnect_interval: 5                                                               
-
-                            hash_salt: auto                                                                            
-
-                            require_client_auth: true                                                                  
-
-                            use_tls: true                                                                              
-
-                        template: openfl.federation.Network                                                          
-
-                        task_runner:                                                                                   
-
-                        settings: {}                                                                                 
-
-                        template: src.taskrunner.TemplateTaskRunner                                                  
-
-                        tasks:                                                                                         
-
-                        aggregated_model_validation:                                                                 
-
-                            function: validate_task                                                                    
-
-                            kwargs:                                                                                    
-
-                            apply: global                                                                            
+All the evaluation run overrides like round_number check skipping, defaulting round_number to 1 and task_group selection are all now baked in aggregator behavior and no manual edits are needed to switch b/w `learning` and `evaluation` run of a plan.
 
-                            metrics:                                                                                 
+Requirements
+------------
+- OpenFL (version ≥ 1.8 recommended)
+- A pre-trained model (``best.pbuf``) generated from a training run

```

Comment:
Does the model file need to be called `best.pbuf` specifically, or is it configurable? Could you further specify where this file should be located?